StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -28,12 +28,18 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
 
     // MARK: Private properties
     private let configuration: PaymentSheet.Configuration
-    private let isCBCEligible: Bool
-    private let paymentMethodRemove: Bool
-    private let ephemeralKeySecret: String
+    private let intent: Intent
 
     private var updateViewController: UpdateCardViewController?
 
+    private var paymentMethodRemove: Bool {
+        intent.elementsSession.allowsRemovalOfPaymentMethodsForPaymentSheet()
+    }
+
+    private var isCBCEligible: Bool {
+        intent.cardBrandChoiceEligible
+    }
```

Comment:
nit: These aren't really computed either, are they?  Should they be `let`s? Computed vars implies the value can change, which is harder to reason about compared to `let`.